Mac/BuildScript/build-installer.py

```diff
@@ -359,9 +359,9 @@ def library_recipes():
                   ),
           ),
           dict(
-              name="SQLite 3.45.1",
-              url="https://sqlite.org/2024/sqlite-autoconf-3450100.tar.gz",
-              checksum="cd9c27841b7a5932c9897651e20b86c701dd740556989b01ca596fcfa3d49a0a",
+              name="SQLite 3.45.3",
+              url="https://sqlite.org/2024/sqlite-autoconf-3450300.tar.gz",
+              checksum="b2809ca53124c19c60f42bf627736eae011afdcc205bb48270a5ee9a38191531",
```

Comment:
Perhaps we should start using SHA3-256 so _our_ checksum matches the SQLite download page checksum. cc. @SethMichaelLarson